pkg/job_controller/job.go

```diff
@@ -251,6 +251,11 @@ func (jc *JobController) ReconcileJobs(
 		return result, err
 	}
 
+	// Metring first pod launch delay when job state transit from created to running.
+	if !commonutil.IsRunning(*oldStatus) && commonutil.IsRunning(jobStatus) {
```

Comment:
The comment says from "created" , but the code says !isRunning, 
the job could be transitioned from Restarting too

Reply:
done, the old Status must be `Created`.